foreign_cc/private/framework/toolchain.bzl

```diff
@@ -96,7 +89,7 @@ def register_framework_toolchains(register_toolchains = True):
 
     for item in TOOLCHAIN_MAPPINGS:
         # Generate a toolchain name without the `.bzl` suffix
-        toolchain_name = "rules_foreign_cc_framework_toolchain_" + item.file.name[:-len(".bzl")]
+        toolchain_name = "rules_foreign_cc_framework_toolchain_" + item.file.split(":")[1][:-len(".bzl")]
```

Comment:
Always nervous about raw indexing. Can you update the comment to be a bit more descriptive about that?

Reply:
I think it's fine just want to make sure the expectation is super clear as indexing is easy to miss.

I think it's probably time to discuss writing a process wrapper instead of making more changes to the toolchain. If you could assign `item.file.split(":")[1][:-len(".bzl")]` to another value with some comment I'd be happy.